clinics.js

```diff
@@ -223,136 +258,228 @@ module.exports = function (common) {
+      common.assertArgumentsSize(3);
+      common.doPostWithToken(
+        `/v1/clinics/${clinicId}/invites/clinicians`,
+        clinician,
+        { 200: function(res){ return res.body; }, 404: [] },
+        cb
+      )
+    },
+
+    /**
+     * Resend the clinician invite
+     *
+     * @param {String} clinicId - clinic Id
+     * @param {String} inviteId - invite Id
+     * @param {Function} cb
+     */
+    resendClinicianInvite: function(clinicId, inviteId, cb){
+      common.assertArgumentsSize(3);
+      common.doPatchWithToken(
+        `/v1/clinics/${clinicId}/invites/clinicians/${inviteId}`,
+        '',
+        { 200: function(res){ return res.body; }, 404: [] },
+        cb
+      )
+    },
+
+    /**
+     * Delete the outgoing clinician invite
+     *
+     * @param {String} clinicId - clinic Id
+     * @param {String} inviteId - invite Id
      * @param {Function} cb
-     * @returns {cb} cb(err, response)
      */
-    deletePatientFromClinic: function(clinicId, patientId, cb){
+    deleteClinicianInvite: function(clinicId, inviteId, cb){
       common.assertArgumentsSize(3);
       common.doDeleteWithToken(
-        `/clinics/${clinicId}/patients/${patientId}`,
+        `/v1/clinics/${clinicId}/invites/clinicians/${inviteId}`,
         { 200: function(res){ return res.body; }, 404: [] },
         cb
-      );
+      )
     },
 
     /**
-     * Retrieve a list of clinicians for a clinic
+     * Retrieve list of patient invitations for clinic
      *
-     * @param {Number} clinicId - Id of the clinic
+     * @param {String} clinicId - Id of the clinic
      * @param {Function} cb
      * @returns {cb} cb(err, response)
      */
-    getCliniciansFromClinic: function(clinicId, cb){
+    getPatientInvites: function(clinicId, cb){
       common.assertArgumentsSize(2);
       common.doGetWithToken(
-        `/clinics/${clinicId}/clinicians`,
+        `/v1/clinics/${clinicId}/invites/patients`,
         { 200: function(res){ return res.body; }, 404: [] },
         cb
       );
     },
 
     /**
-     * Add a clinician to a clinic
+     * Accept a pending invite from patient user
      *
-     * @param {Number} clinicId - Id of the clinic
-     * @param {Object} clinician
-     * @param {String} [clinician.clinicianId] - Id of the clinician
-     * @param {String} [clinician.clinicId] - Id of the clinic
-     * @param {String[]} [clinician.permissions] - Array of string permissions
+     * @param {String} clinicId - Id of the clinic
+     * @param {String} inviteId - Id of the invite
      * @param {Function} cb
      * @returns {cb} cb(err, response)
      */
-    addClinicianToClinic: function(clinicId, clinician, cb){
+    acceptPatientInvitation: function(clinicId, inviteId, cb){
       common.assertArgumentsSize(3);
-      common.doPostWithToken(
-        `/clinics/${clinicId}/clinicians`,
-        clinician,
+      common.doPutWithToken(
+        `/v1/clinics/${clinicId}/invites/patients/${inviteId}`,
+        '',
```

Comment:
I'm curious why we're passing an empty arg here.  I'm assuming the backend has a default, but allows this parameter to contain other values.

Reply:
The API spec details that this endpoint doesn't expect anything for the `body` of the `PUT`, but the format of the `doPutWithToken` essentially requires a `data` that will be sent to the server in order to not stomp on the `codes`/`cb` params. Do you think that going with `null` as opposed to the empty string would be better/more explicit (like `confirm.removeInvite`)?

Reply:
Using `null` does seem more fitting to me.